GameServerLib/Logic/DependencyInjection/DI.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Ninject;
+
+namespace LeagueSandbox.GameServer.Logic.DependencyInjection
+{
+    public static class DI
```

Comment:
I don't know if we should have a static class in the first place here, beats me. Personally I stay away from static unless they're absolutely necessary